proofs/eo/cpc/expert/CpcExpert.eo

```diff
@@ -37,22 +37,20 @@
 (declare-parameterized-const @shared_selector ((D Type :opaque) (T Type :opaque) (n Int :opaque)) (-> D T))
 
 
-; define: $get_aci_normal_form_expert
+; program: $get_aci_normal_form_expert
 ; args:
 ; - t S: The term to compute the normal form for.
 ; return: >
 ;   The result of normalizing t for the proof rule aci_norm. In
 ;   particular, if t is an application of a known (expert) operator that has the property
 ;   of being either associative and commutative (resp. associative) we call
 ;   the method $get_aci_norm (resp. $get_a_norm).
-(define $get_aci_normal_form_expert ((S Type :implicit) (t S))
-  (eo::match ((x1 Bool) (x2 Bool :list) (m Int) (xf1 (FiniteField m)) (xf2 (FiniteField m) :list))
-    t
-    (
-      ((sep x1 x2)       (@aci_sorted sep ($get_ai_norm t)))
-      ((ff.add xf1 xf2)  (@aci_sorted ff.add ($get_ai_norm t)))
-      ((ff.mul xf1 xf2)  (@aci_sorted ff.mul ($get_ai_norm t)))
-    )
+(program $get_aci_normal_form_expert ((T Type) (x1 Bool) (x2 Bool :list) (m Int) (xf1 (FiniteField m)) (xf2 (FiniteField m) :list))
+  (T) T
+  (
+    (($get_aci_normal_form_expert (sep x1 x2))       (@aci_sorted sep ($get_a_norm (sep x1 x2))))
+    (($get_aci_normal_form_expert (ff.add xf1 xf2))  (@aci_sorted ff.add ($get_a_norm (ff.add xf1 xf2))))
+    (($get_aci_normal_form_expert (ff.mul xf1 xf2))  (@aci_sorted ff.mul ($get_a_norm (ff.mul xf1 xf2))))
```

Comment:
The old documentation says `$get_a_norm`, but old code uses `$get_ai_norm`. Which is right?